data/templates/get_driveid.js

```diff
@@ -189,14 +189,12 @@ function parseVdInfo(pathList) {
     });
 
     return pciLines.map(function(line) {
-        var scsiId = line[1].slice(line[1].lastIndexOf('-') + 1), vdStr;
+        var scsiId = line[1].slice(line[1].lastIndexOf('-') + 1),
```

Comment:
Could you please paste some example line here, so that it helps to understand your code.